src/logic/utils.ts

```diff
@@ -1,14 +1,28 @@
 import {
     NEW_PLAYER_HIGHLIGHT_DAYS,
+    NUMBER_OF_PLAYERS_FOR_VALID_MATCH,
+    PLAYER_MAXIMUM_GAMES_AS_NEW_PLAYER,
     PLAYER_MINIMUM_GAMES_REQUIRED,
     PLAYER_MINIMUM_WINS_REQUIRED
 } from "../components/constants";
 import { Player } from "../types/domain/Player";
 
+
+/**
+ * Gets the win rate as a percentage (number, without the "%").
+ * @param winCount Wins - most likely valid wins.
```

Comment:
in both of these instances, why is there uncertainty ("most likely") ? :) 

In this case, we would say something like "the number of wins". It is up to the caller to decide if they want to use valid wins or not. As the creator of the function, it is not the responsibility of us to ensure that they are "valid wins :)

Reply:
aye, i fixed this a bit. problem: i want to give a reminder to the user not to use matches.length if it's not appropriate. how should i word it better?

Reply:
that's a good idea to remind the user not to use matches.length, but as I said prior, unfortunately, that is out of the responsibility of this method. :(

The user may be wanting to take into account invalid matches in their calculation, and we cannot make that assumption. This is one of those instances that we can stop folks from hurting themselves. Hence, my answer here is that we don't provide the warning. 

However, if you REALLY wanted to, you could maybe in the description of function say something like, when calling this method, ensure that you are comparing the winrate to the correct total count of matches, (which may or may not include invalid matches). I would probably say that what may happen is that someone else who is in the repo may see this comment later on, see it as too verbose/unnecessary, and remove it :)

Reply:
fixed the wording to:
@param totalCount Number of matches to consider - use valid matches when appropriate.